server/server.go

```diff
@@ -536,6 +536,11 @@ func (s *Server) SetScheduleConfig(cfg ScheduleConfig) error {
 	old := s.scheduleOpt.load()
 	s.scheduleOpt.store(&cfg)
 	if err := s.scheduleOpt.persist(s.kv); err != nil {
+		s.scheduleOpt.store(old)
+		log.Error("schedule config updated failed",
```

Comment:
How about changing the error message to `failed to update xxx`?

Reply:
Sure. :ok_hand: